azurerm/internal/services/compute/linux_virtual_machine_scale_set_resource.go

```diff
@@ -126,6 +126,11 @@ func resourceArmLinuxVirtualMachineScaleSet() *schema.Resource {
 				Default:  false,
 			},
 
+			"encryption_at_host": {
```

Comment:
Also I suppose we could rename this to `encryption_at_host_enabled` given that this is a switch, what do you think?

Reply:
I've just addressed this feedback 👌